tools/jdbc/src/main/java/org/duckdb/DuckDBPreparedStatement.java

```diff
@@ -44,6 +44,7 @@ public class DuckDBPreparedStatement implements PreparedStatement {
 	private boolean returnsChangedRows = false;
 	private boolean returnsNothing = false;
 	private boolean returnsResultSet = false;
+	boolean closeOnCompletion;
```

Comment:
Should this be `private boolean closeOnCompletion = false;` to match the other flags here?